card/src/main/res/values/strings.xml

```diff
@@ -11,16 +11,16 @@
     <string name="checkout_card_holder_hint">Name on card</string>
     <string name="checkout_card_expiry_date_hint">Expiry date</string>
     <string name="checkout_card_expiry_date_optional_hint">Expiry date (optional)</string>
-    <string name="checkout_card_security_code_hint">CVC / CVV</string>
-    <string name="checkout_card_security_code_optional_hint">CVC / CVV (optional)</string>
+    <string name="checkout_card_security_code_hint">Security code</string>
```

Comment:
There's quite a few changes in this PR, i think we need release notes especially for the more prominent changes like this one

Reply:
I just added the list of updated string resources